minio/015-minio-secret.yaml

```diff
@@ -7,6 +7,6 @@ metadata:
     app: minio
 type: Opaque
 stringData:
-  MINIO_ROOT_USER: # to be replaced by minio-user
-  MINIO_ROOT_PASSWORD: # to be replaced by minio-password
+  MINIO_ROOT_USER:  #
```

Comment:
Probably we should add some info about completing this file before deploying MinIO (e.g. in a README.md)

Reply:
We can write a README for the Minio part with the basic instructions to run in k8s and to explain credential parts.